toolkit/tools/pkg/imagecustomizerlib/customizepackages.go

```diff
@@ -82,6 +82,11 @@ func addRemoveAndUpdatePackages(buildDir string, baseConfigPath string, config *
 		}
 	}
 
+	err = cleanTdnfCache(imageChroot)
```

Comment:
It would be good to update one (or more) of the `TestCustomizeImagePackages*` tests to verify the tdnf cache is in fact empty.

Reply:
Updated the `TestCustomizeImagePackagesUpdate()` unit test to verify this as it is the most appropriate unit test here.